docs/manual-EN/server-administration/storage-balance.md

```diff
@@ -0,0 +1,219 @@
+# Storage Balance Usage
+Nebula's services are composed of  three parts: graphd, storaged and metad. The **balance** in this document targets the storaged. Currently, storage is scaled by `balance` prodecure. There are two kinds of balance command, one is to move data, the command is `balance data`; the other one only changes the distribution of leader partition to balance load without moving data, the command is `balance leader`.
+
+## Balance data
+Let's use an example to expand the cluster from 3 storage instances to 8 to show how to balance data.
+### Step 1 Prerequisites
+Deploy a cluster with three replicas, one graphd, one metad and three storaged. Details on deployment please refer [Cluster Deploy](). Check the status of the deployed cluster using command `SHOW HOSTS`.
+#### Step 1.1
+```
+(user@127.0.0.1) [(none)]> SHOW HOSTS
+
+================================================================================================
+| Ip            | Port  | Status | Leader count | Leader distribution | Partition distribution |
+================================================================================================
+| 192.168.8.210 | 34600 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34700 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+| 192.168.8.210 | 34500 | online | 0            | No valid partition  | No valid partition     |
+------------------------------------------------------------------------------------------------
+Got 3 rows (Time spent: 5886/6835 us)
+```
+
+**_Explanations on the returned results:_**
+
+- **IP** and **Port** is the present storage instance, the cluster started three storaged instances without any data. (192.168.8.210:34600, 192.168.8.210:34700, 192.168.8.210:34500)
```

Comment:
the cluster starts with three storaged instances (192.168.8.210:34600, 192.168.8.210:34700, 192.168.8.210:34500) without any data.